common/src/service/HttpClientHelper.cs

```diff
@@ -50,6 +51,7 @@ internal sealed class HttpClientHelper : IHttpClientHelper
             _preRequestActionForAllRequests = preRequestActionForAllRequests;
             _customHttpProxy = customHttpProxy;
             TlsVersions.Instance.SetLegacyAcceptableVersions();
+            _certificateRevocationCheck = TlsVersions.Instance.CertificateRevocationCheck;
```

Comment:
You can cut out this extra variable, and just assign TlsVersions.Instance.CertificateRevocationCheck directly on line 868.

This way if the value is changed after the fact, they don't get the cached value.